### PR TITLE
[Mercure] Update docs for v0.11

### PR DESCRIPTION
**Pull request by dunglas**
The configuration changed in the upcoming version 0.11 (currently in beta).

### CODE REVIEW

File: mercure.rst [hunk 1]
```diff
@@ -68,11 +68,17 @@ clients.
...
 .. code-block:: terminal
 
-    $ ./mercure --jwt-key='!ChangeMe!' --addr='localhost:3000' --allow-anonymous --cors-allowed-origins='*'
+    SERVER_NAME=:3000 MERCURE_PUBLISHER_JWT_KEY="!ChangeMe!" MERCURE_SUBSCRIBER_JWT_KEY="!ChangeMe!" ./mercure run -config Caddyfile.dev
+
+On Windows run:
+
+.. code-block:: terminal
+
+    $envSERVER_NAME=':3000'; $env:MERCURE_PUBLISHER_JWT_KEY='!ChangeMe!'; $env:MERCURE_SUBSCRIBER_JWT_KEY='!ChangeMe!'; .\mercure.exe run -config Caddyfile.dev
```
Comment (OskarStark) [inline mercure.rst]:
```suggestion
    $env:SERVER_NAME=':3000'; $env:MERCURE_PUBLISHER_JWT_KEY='!ChangeMe!'; $env:MERCURE_SUBSCRIBER_JWT_KEY='!ChangeMe!'; .\mercure.exe run -config Caddyfile.dev
```
?

File: mercure.rst [hunk 2]
```diff
@@ -68,11 +68,17 @@ clients.
 An official and open source (AGPL) implementation of a Hub can be downloaded
 as a static binary from `Mercure.rocks`_.
 
-Run the following command to start it:
+On Linux and Mac, run the following command to start it:
 
 .. code-block:: terminal
 
-    $ ./mercure --jwt-key='!ChangeMe!' --addr='localhost:3000' --allow-anonymous --cors-allowed-origins='*'
+    SERVER_NAME=:3000 MERCURE_PUBLISHER_JWT_KEY="!ChangeMe!" MERCURE_SUBSCRIBER_JWT_KEY="!ChangeMe!" ./mercure run -config Caddyfile.dev
```
Comment (OskarStark) [inline mercure.rst]: -config or --config?

Same question for windows

Comment (dunglas) [reply mercure.rst]: `-config` (it's a Caddy server option).

File: mercure.rst (hunk 1 above)
Comment (dunglas) [reply mercure.rst]: Good catch!

File: mercure.rst [hunk 3]
```diff
@@ -68,11 +68,17 @@ clients.
 An official and open source (AGPL) implementation of a Hub can be downloaded
 as a static binary from `Mercure.rocks`_.
 
-Run the following command to start it:
+On Linux and Mac, run the following command to start it:
 
 .. code-block:: terminal
```
Comment (wouterj) [inline mercure.rst]:
```suggestion
.. rst-class:: command-linux
.. code-block:: terminal
```

File: mercure.rst (hunk 2 above)
Comment (wouterj) [inline mercure.rst]:
```suggestion
    $ SERVER_NAME=:3000 MERCURE_PUBLISHER_JWT_KEY="!ChangeMe!" MERCURE_SUBSCRIBER_JWT_KEY="!ChangeMe!" ./mercure run -config Caddyfile.dev
```

File: mercure.rst [hunk 4]
```diff
@@ -68,11 +68,17 @@ clients.
...
 
-    $ ./mercure --jwt-key='!ChangeMe!' --addr='localhost:3000' --allow-anonymous --cors-allowed-origins='*'
+    SERVER_NAME=:3000 MERCURE_PUBLISHER_JWT_KEY="!ChangeMe!" MERCURE_SUBSCRIBER_JWT_KEY="!ChangeMe!" ./mercure run -config Caddyfile.dev
+
+On Windows run:
+
+.. code-block:: terminal
+
+    $env:SERVER_NAME=':3000'; $env:MERCURE_PUBLISHER_JWT_KEY='!ChangeMe!'; $env:MERCURE_SUBSCRIBER_JWT_KEY='!ChangeMe!'; .\mercure.exe run -config Caddyfile.dev
```
Comment (wouterj) [inline mercure.rst]:
```suggestion
.. rst-class: command-windows
.. code-block:: terminal

    > $env:SERVER_NAME=':3000'; $env:MERCURE_PUBLISHER_JWT_KEY='!ChangeMe!'; $env:MERCURE_SUBSCRIBER_JWT_KEY='!ChangeMe!'; .\mercure.exe run -config Caddyfile.dev
```

**Review by wouterj — COMMENTED** [review]
Thank you! I've left 3 minor style comments.

I'm wondering where `Caddyfile.dev` is coming from? Do I have to define it locally? Is it an implementation detail of a file somewhere internally in Mercure?

**Comment by dunglas** [comment]
`Caddyfile.dev` is included in the archive as well as in the Docker image.

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
v0.11 was released three days ago (https://github.com/dunglas/mercure/releases/tag/v0.11.0) so let's merge this. Kévin, congrats on the release!

**Pull request merged**